df.art.xml

```diff
@@ -343,7 +343,7 @@
         <compound name='name' type-name='language_name'/>
         <int32_t name='originating_entity' ref-target='historical_entity'/> "originating in ..."
         <int32_t name='original_author' ref-target='historical_figure'/> "originally devised by ..."
-        <int32_t/>
+        <int32_t name='subject_hf' ref-target='historical_figure'/>
```

Comment:
Liked the ref-target attribute!

Reply:
Yes, it ought to be used wherever it's appropriate.